models/repo_list.go

```diff
@@ -249,3 +249,28 @@ func SearchRepositoryByName(opts *SearchRepoOptions) (RepositoryList, int64, err
 
 	return repos, count, nil
 }
+
+// FindUserAccessiableRepoIDs find all accessiable repositories' ID by user's id
+func FindUserAccessiableRepoIDs(userID int64) ([]int64, error) {
```

Comment:
does this include organization repos that could be accessed by the user?

Reply:
And a typo in **accessible**?

Reply:
@JonasFranzDEV yes, included.
@mbger done.